### DISCUSSION TITLE
BIG Issues come unexpectedly...

### DISCUSSION THREAD

**Discussion by daudix** [GradienceTeam/Gradience#111] [General]
Shortly after publishing AdwCustomizer in TWIG, I recieved this message from Tobias Bernard (Purism and GNOME designer)

![image](https://user-images.githubusercontent.com/77155297/184263481-41d21543-7ced-4415-b849-6f4a8f9f5757.png)

What we can do in this situation?

_This can ruin all publishing plans, and there is a whole lot of things we need to rebrand..._

**Comment by 0xMRTT** [comment]
There is gdm settings

**Reply by daudix** [reply]
I immideatly thought about that, double standarts?

**Comment by 0xMRTT** [comment]
gnome customizer

**Reply by daudix** [reply]
This breaks [GNOME HIG](https://developer.gnome.org/hig/guidelines/app-naming.html)

**Comment by 0xMRTT** [comment]
Maybe if in the welcome setup we add a page to specify that the app isn't official and same in README

**Reply by daudix** [reply]
This is the easiest solution.

_And the only way to keep very nice name_

**Reply by 0xMRTT** [reply]
a dialog would be great, but i think it's important that it not use the name adwaita directly

@ tobias bernard

**Comment by 0xMRTT** [comment]
Otherwise we need to brainstorm as specified in https://developer.gnome.org/hig/guidelines/app-naming.html

**Reply by daudix** [reply]
For start

Adwaita - Window
Window - Controls, Toggles
Toggles - Switcher
Switch - Change
Theme - Look

**Comment by 0xMRTT** [comment]
not sure i'm aware of that app, but gdm is a bit different because it's not a brand in the way adwaita is
it's the name of the design system, and pretty core to the developer experience
yeah, the word means non-duality in sanskrit
could call it "Multiplicity" or something like that :)

> multiplicity manager

well with that point the manager part might not even be needed :D

**Reply by daudix** [reply]
Consistency Lab/Studio/Manager?

**Reply by 0xMRTT** [reply]
Consistency manager seems to be great 🎉😎

**Reply by daudix** [reply]
For now we can add a warning in readme, but in v0.2.0 we can change the name

**Reply by 0xMRTT** [reply]
Will work on tomorrow because need to change a lot of things. 

The codename will stay AdwCustomizer. And the app can still be called AdwCustomizer like nautilus and files

**Reply by daudix** [reply]
Agree

**Reply by 0xMRTT** [reply]
You can edit the TWIG report and specify the name changement.

**Reply by daudix** [reply]
But I will _Freaking_ miss the Adwaita Manager.

**Reply by 0xMRTT** [reply]
Me too 😭

**Reply by daudix** [reply]
Added to TWIG:

Note: Adwaita Manager is NOT an official GNOME app, the name will change in v0.2.0

**Comment by 0xMRTT** [comment]
except that it's a bit long and might get truncated in the shell, but that's a different issue :D

**Reply by daudix** [reply]
Cons Lab?

_Sounds more like a current situation_

**Reply by 0xMRTT** [reply]
Multiplicity

**Reply by 0xMRTT** [reply]
Because it's the translation of Adwaita and there is multiple options and in the future multiple plugins 😎

**Reply by daudix** [reply]
Sounds good, but it hard to get it without knowing translation

**Reply by daudix** [reply]
[Vikartana](https://sanskritdictionary.com/vikartana/209617/1)?

**Reply by 0xMRTT** [reply]
We can choose to not translate the name ?

**Reply by daudix** [reply]
As you want, brainstorm is brainstorm

**Reply by 0xMRTT** [reply]
> [Vikartana](https://sanskritdictionary.com/vikartana/209617/1)?

Gøød. It's short, not problem with hig.

**Reply by daudix** [reply]
What about [Advaidha](https://sanskritdictionary.com/advaidha/4675/1)?

**Reply by 0xMRTT** [reply]
> Multiplicity

Множественность

**Reply by daudix** [reply]
Да, еще как умножение

**Reply by 0xMRTT** [reply]
It was just Google translator 😅

**Reply by 0xMRTT** [reply]
> What about [Advaidha](https://sanskritdictionary.com/advaidha/4675/1)?

Too complex

**Reply by daudix** [reply]
But if you pronouce it, it sound like Adwaita

**Reply by 0xMRTT** [reply]
Sounds good

**Reply by 0xMRTT** [reply]
The best one for now
🎉

**Comment by daudix** [comment]
Paint Bucket / Roller?

**Reply by 0xMRTT** [reply]
Paint bucket sounds goood

**Comment by 0xMRTT** [comment]
Coloration

**Reply by daudix** [reply]
Color Accent?

**Reply by 0xMRTT** [reply]
Too specific

**Reply by daudix** [reply]
Colorizer?

**Comment by daudix** [comment]
Polarizer?

**Reply by 0xMRTT** [reply]
Looks like pictures

**Reply by 0xMRTT** [reply]
I think that's more a name for a camera app

**Reply by daudix** [reply]
Displays have it too

**Comment by 0xMRTT** [comment]
Personalizer

**Comment by 0xMRTT** [comment]
Or simply: Customizer

**Reply by 0xMRTT** [reply]
Come back to the first name AdwCustomizer

**Reply by 0xMRTT** [reply]
If think that just Customizer in great

**Comment by daudix** [comment]
@0xMRTT As name now unrealted to the app icon, is there need to update it?

**Reply by 0xMRTT** [reply]
Yes

**Reply by daudix** [reply]
Well... OK

**Reply by 0xMRTT** [reply]
See #117
